website/www/site/content/en/get-started/quickstart-py.md

```diff
@@ -23,7 +23,7 @@ If you're interested in contributing to the Apache Beam Python codebase, see the
 
 {{< toc >}}
 
-The Python SDK supports Python 2.7, 3.5, 3.6, and 3.7. New Python SDK releases will stop supporting Python 2.7 in 2020 ([BEAM-8371](https://issues.apache.org/jira/browse/BEAM-8371)). For best results, use Beam with Python 3.
+The Python SDK supports Python 2.7, 3.6, and 3.7. New Python SDK releases will stop supporting Python 2.7 in 2020 ([BEAM-8371](https://issues.apache.org/jira/browse/BEAM-8371)). For best results, use Beam with Python 3.
```

Comment:
```suggestion
The Python SDK supports Python 3.6, 3.7, and 3.8. Beam 2.24.0 was the last release with support for Python 2.7 and 3.5.
```
We dropped support for 2.7 and 3.5 at the same time, and we now have support for 3.8.

CC @tvalentyn - any thoughts on how we can make sure this stays up to date in the future?

Reply:
Thanks, all. We have a plan to put together a guide for what needs to be updated in https://issues.apache.org/jira/browse/BEAM-10605. Added a note there.